### ISSUE TITLE
Add classic topomap NGI 2000

### ISSUE THREAD

**Issue by jbelien** [osmbe/play#5]
From @StevenClays (#4):
> Is it possible to add the topomap (2000) to the historical maps?
https://topomapviewer.ngi.be/?l=nl&x=629911.03&y=660643.61&zoom=8&baseLayer=top10r

**Comment by jbelien** [comment]
Hello @StevenClays,
The link you sent (NGI Topo10) seems only to be compatible with [`EPSG:3812`](https://epsg.io/3812) and [`31370`](https://epsg.io/31370).
We need it to be compatible with [`EPSG:3857`](https://epsg.io/3857) to be added to our historical maps.

I looked on IGN/NGI website and couldn't find it.
If you find it (anywhere), please provide the link.
Thanks !

**Comment by StevenClays** [comment]
No possibilities for conversion then? @mstuyts?

**Comment by mstuyts** [comment]
For the NGI Topo10 layer, I have only found tiles in EPSG:3812.  @jbelien, what's the url for tiles in EPSG:31370? I don't think the tiles can be converted to EPSG:3857, because they are prerendered. I looked into using MapProxy to reproject the tiles, because I use that to reproject WMS services.  But I haven't found a way to that for tile map layers like the NGI Topo10 one.

I haven't been able to view these tiles outside the NGI TopoMapViewer. I don't know if it's possible in Leaflet to set the CRS of tile layer that's not a WMS source. 

All other levels Topo25, Topo50, Topo100, Topo250 and Topo300 are available via WMS: https://wms.ngi.be/inspire/topomaps/service and they have EPSG:3857 and they can be added to Leaflet.

Another related detailed NGI layer is CartoWeb and that's available via WMTS: https://www.ngi.be/cartoweb/1.0.0/WMTSCapabilities.xml that also has EPSG:3857 available.

**Comment by StevenClays** [comment]
To explain: topographic maps of the 1990'ies-2000'ies are actually historical maps. So it would be nice to add them. Topo25 & Topo50 would allready be great!

**Comment by joostschouppe** [comment]
Added in https://github.com/osmbe/play/commit/97a6be3af59d09d7f447bb5236334d63f89199e7
Remains to be seen if it actually works :)

**Comment by mstuyts** [comment]
For some reason, tiles are missing for some areas.  
![ngi_topo25_tiles](https://user-images.githubusercontent.com/11028014/145214643-dfb9da36-15bc-430c-bf7b-f33fb3bab932.png)

I checked the same service in QGIS, there it shows the same issue, so I guess the wms service has an issue.

**Comment by joostschouppe** [comment]
Yes, we noticed as well! I happen to have met someone from NGI at the Trefdag and mentioned it. He contacted the right colleagues
Now it would also be nice to make a smarter implementation that shows different maps at different zoomlevels.

**Comment by joostschouppe** [comment]
So... the service is NOT the 2000 version as I understood but rather the most current one! The holes are because there is no update yet! Everything you see is from 2016 and more recent, the empty areas should be filled in by 2022. So we should remove this version; the search for a 2000 version is still on...
